hub/constraints/instruction-handling/create/constraints/unexceptional.lisp

```diff
@@ -63,6 +63,19 @@
                                     (eq!    (shift    account/RLPADDR_KEC_LO       CREATE_first_creator_account_row___row_offset)     (create-instruction---init-code-hash-lo))
                                     )))
 
-(defun    (create-instruction---address-computation-recipe)    (+    (*    RLP_ADDR_RECIPE_1    (create-instruction---is-CREATE))
-                                                                     (*    RLP_ADDR_RECIPE_2    (create-instruction---is-CREATE2))
-                                                                     ))
+(defun    (create-instruction---address-computation-recipe)    (+    (*   (create-instruction---is-CREATE)    RLP_ADDR_RECIPE_1)
+                                                                     (*   (create-instruction---is-CREATE2)   RLP_ADDR_RECIPE_2)))
+
+(defun    (create-instruction---init-code-hash-hi)    (*    (create-instruction---is-CREATE2)
```

Comment:
Why using `*` rather than nested `if-not-zero`? Same comment for the constraint below.

Reply:
It's the same in the end. In the spec it's written as a product like here.